crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -94,13 +93,16 @@ struct ProviderBindgenConfig {
     pub(crate) exposed_interface_deny_list: Vec<LatticeExposedInterface>,
 
     /// wit-bindgen configuration that is passed straight through (uses vendored wit-bindgen)
-    pub(crate) wit_bindgen_cfg: WitBindgenConfig,
+    pub(crate) wit_bindgen_cfg: Option<WitBindgenConfig>,
```

Comment:
Suggestion: I think the comment [here](https://github.com/wasmCloud/wasmCloud/pull/963/files#diff-c35e6db0016f48f0c6fe14869e51b1fbffd9c6d8db1be7933222b5d519e6045aR922) would actually be better if it were here (assuming I've understood correctly)

Reply:
Sure, will move this!

- [ ] move docs on `wit_bindgen_cfg`